test/kata5.reachDestination.test.js

```diff
@@ -1,5 +1,7 @@
 const { reachDestination } = require("../src");
 
 describe("reachDestination", () => {
-  test("returns string with estimated time of arrival", () => {});
+  test("returns string with estimated time of arrival", () => {
+    expect(reachDestination(44, 10)).toEqual("I should be there in 4.5 hours.");
```

Comment:
A few more examples would be good here to verify some extra cases. Also what would happen if invalid values are provided?